src/utils/miscellaneous.ts

```diff
@@ -265,4 +265,4 @@ export const isAnOrderId = (text: string): boolean => text.match(/^0x[a-fA-F0-9]
  *
  * @param text Possible address string to check
  */
-export const isAnAddressAccount = (text: string): boolean => text.match(/^0x[a-fA-F0-9]{40}$/)?.input !== undefined
+export const isAnAddressAccount = (text: string): boolean => text.match(/^0x[a-fA-F0-9]{40}$|[a-zA-Z0-9]+\.[a-zA-Z]+$/)?.input !== undefined
```

Comment:
Yep, my bad.  I added a check in the callback hook that runs the regex again for just the second half (i.e. `[a-zA-Z0-9]+\.[a-zA-Z]+$/`) to see if it looks like a domain name since an ETH address will never have a period in it so this bypasses the network check when it's just a normal 0x... address.